xoinvader/background.py

```diff
@@ -0,0 +1,261 @@
+from xoinvader.common import Settings
+from xoinvader.utils import Point, Surface
+from xoinvader.render import Renderable
+
+
+class Chunk(object):
+    """Class for storing background chunks.
+
+    :param name: name of the chunk
+    :type name: string
+
+    :param lines: list of all chunk lines
+    :type lines: list
+
+    :param length: length of lines list (for convenience)
+    :type length: integer
+
+    :param __MAGIC__: magic of chunk boundary in the file
+    :type __MAGIC__: string
+    """
+
+    __MAGIC__ = "~chunk~"
+
+    def __init__(self, name):
+        self.name = name
+        self.lines = []
+        self.length = 0
+
+    def add_line(self, line):
+        """Add line to `lines` list of the chunk. Increases `length` by 1.
+
+        :param line: line to add
+        :type line: string
+        """
+        self.lines.append(line)
+        self.length += 1
+
+    def __getitem__(self, index):
+        return self.lines[index]
+
+    def __setitem__(self, index, item):
+        self.lines[index] = item
+
+
+def load_chunks(filename, trim_width=None):
+    """Load chunks from file.
+
+    File format:
+    ```
+    -~chunk~- <chunk_name>
+    <data>
+    ...
+    ```
+
+    `~chunk~` is magic string, and chunk spans until next magic line, or
+    until end of file.
+
+    Completely blank lines are ignored.
+    If `trim_width` is provided, all lines are truncated to this length.
+
+    :return: list of loaded chunks
+    :rtype: list
+
+    :param filename: path to file to load chunks from
+    :type filename: string
+
+    :param trim_width: width to truncate long lines to
+    :type trim_width: integer"""
+
+    with open(filename, "r") as f:
+        chunks = []
+        current_chunk = None
+        names = []
+        for line in f:
+            line = line.rstrip('\n')
```

Comment:
double quotes please.

PEP8 allows using single and double quotes, but project should use only one of them, for order.